Dockerfile

```diff
@@ -1,16 +1,7 @@
 FROM agapito/centos7-scala:latest
 
-WORKDIR /root
+COPY . /twitter-lm-extractor/
 
-# install deps and create testdir
-RUN yum -y install make && \
-    mkdir -p /root/twitter-lm-extractor
+WORKDIR /twitter-lm-extractor
 
-WORKDIR /root/twitter-lm-extractor
-
-COPY build.sbt /root/twitter-lm-extractor
-COPY src /root/twitter-lm-extractor/src
-COPY project /root/twitter-lm-extractor/project
-
-# run tests
-CMD ["sbt", "clean", "test"]
+ENTRYPOINT ["sbt", "clean", "test"]
```

Comment:
The makefile is welcome. But since we are calling the tests through the entry point directive, the makefile it's not particularly necessary for this particular feature.

Would be nice to have some make commands to run unit tests locally - even though the IDE does that.